emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
+             "user_id" : ecwb.WrapperBase.Access.WORM,
+             "common_place_id" : ecwb.WrapperBase.Access.WORM
```

Comment:
I was trying to make it in a way I could recreate it

On Friday, February 5, 2016, shankari <notifications@github.com> wrote:

> In emission/core/wrapper/common_place.py
> <https://github.com/e-mission/e-mission-server/pull/227#discussion_r52097919>
> :
>
> > @@ -0,0 +1,16 @@
> > +import logging
> > +import emission.core.wrapper.wrapperbase as ecwb
> > +
> > +class CommonPlace(ecwb.WrapperBase):
> > +    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
> > +             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
> > +             "user_id" : ecwb.WrapperBase.Access.WORM,
> > +             "common_place_id" : ecwb.WrapperBase.Access.WORM
>
> why not just use _id? The good thing about _id is that mongodb will
> autopopulate it. How were you planning on generating a unique ID that would
> be better than mongo's version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/227/files#r52097919>.
>